index.html

```diff
@@ -256,53 +256,112 @@ <h1>Terminology</h1>
+	      Discovery uses a two-phase architecture to resolve the competing requirements to be
+        both open and to restrict access to metadata to authorized entities.
+	      In the first phase, one or more of a set of relatively open
+        "Introduction" mechanisms may be used to generate a set of candidate URLs.  
+        These URLs do not themselves contain metadata,
+	      but are used in the second stage to reference "Exploration"
+        services that can actually provide metadata, after authentication,
+        in the form of <a>Thing Descriptions</a>.
+       </p>
         <figure id="discovery-overview">
             <img src="images/overview.png"
                 srcset="images/overview.svg"
                 class="wot-diagram"
                 alt="Discovery process overview" />
-            <figcaption>Discovery process overview</figcaption>
+            <figcaption>Discovery architecture overview.  
+              A set of open Introduction mechanisms provides a set of URLs, which
+              point at Exploration services that only provide metadata (TDs) after suitable
+              authentication.  Thing Links and Thing Description Directories provide
+              additional flexibility but retrieving further results from these is
+              at the discretion of the application.
+            </figcaption>
         </figure>
-	<p>
-	The WoT discovery process is designed using a two-phase approach.
-	In the first phase, one or more of a set of "Introduction" mechanisms may be used to 
-	generate a set of candidate URLs.  These URLs do not themselves contain metadata,
-	but are used in the second stage, "Exploration", to actually fetch metadata.
-	The intention is that Introductions are relatively open and can support any
-	mechanism that can return a URL, including simply entering URLs directly.
-	Introductions, however, do not include any security or privacy controls and
-	so should not provide metadata directly.  Instead, the provided URLs are followed
-	to "Exploration" mechanisms that will actually provide metadata but allow for
-	suitable security and privacy controls.
+        <p>
+	       The intention is that Introductions are relatively open "first contact"
+         mechanisms to provide a starting point for the rest of the Discovery process.
+         In this document we specify details on several Introduction
+         mechanisms, suitable for different use cases, including both local and non-local
+         scenarios, but Introductions can in fact be provided any mechanism that can return a URL.
+	       Introductions, however, do not include any security or privacy controls and
+	       so should not provide metadata directly. 
+         Instead, the URLs provided by Introductions reference
+	       "Exploration" services.  Exploration services actually do provide metadata,
+         but only after suitable authentication and access controls have been applied.
```

Comment:
Hard to follow. I suggest not shortening "Introduction mechanisms" to "Introductions"
```suggestion
	       The intention is that Introduction mechanisms are relatively open "first contact"
         mechanisms to provide a starting point for the rest of the Discovery process.
         In this document we specify details on several Introduction
         mechanisms, suitable for different use cases, including both local and non-local
         scenarios, but Introductions can in fact be provided any mechanism that can return a URL.
	       Introductions, however, do not include any security or privacy controls and
	       so should not provide metadata directly. 
         Instead, the URLs provided by Introduction mechanisms reference
	       "Exploration" services.  Exploration services actually do provide metadata,
         but only after suitable authentication and access controls have been applied.
```